Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml

```diff
@@ -14,7 +14,7 @@
       d:DesignHeight="1000" d:DesignWidth="2000"
       Title="Runset"  Loaded="Page_loaded">
 
-    <ScrollViewer VerticalScrollBarVisibility="Auto" >
+    <ScrollViewer VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Disabled" >
```

Comment:
why we need this scrollviewer, can this be removed?

Reply:
We removed it